crates/symbolicator/src/services/download/s3.rs

```diff
@@ -187,6 +214,8 @@ mod tests {
     const S3_BUCKET: &str = "symbolicator-test";
 
     fn s3_source_key() -> Option<S3SourceKey> {
+        let aws_credentials_provider =
+            std::env::var("SENTRY_SYMBOLICATOR_TEST_AWS_CREDENTIALS_PROVIDER").ok()?;
```

Comment:
As we are not testing the container provider (which is fair enough, it's up to rusoto to test that) I think getting this from the environment is not needed and it's fine to simply hardcode this.

Reply:
Done.